cmd/influxd/run.go

```diff
@@ -682,6 +669,15 @@ func joinServer(s *influxdb.Server, u url.URL, joinURLs []url.URL) {
 			return
 		}
 	}
+
+	if len(joinURLs) == 0 {
```

Comment:
Not strictly part of this, but are we missing a return at line 666 above? If so, can you add it to this PR?